data/mods/Aftershock/items/afs_tools.json

```diff
@@ -23,6 +23,21 @@
     ],
     "flags": [ "WATCH", "ALARMCLOCK" ]
   },
+  {
+    "id": "afs_atomic_smartphone_flashlight",
+    "copy-from": "afs_atomic_smartphone",
+    "type": "TOOL",
+    "name": "atomic smartphone - Flashlight",
+    "name_plural": "atomic smartphones - Flashlight",
+    "revert_to": "afs_atomic_smartphone",
```

Comment:
Will this chunk of changes fix the following error from where they are trying to get Travis to run on Aftershock?  
09:36:02.947 ERROR : d:\projects\cataclysm-dda-fork\src\json.cpp:112: Failed to visit member 'revert_to' in JsonObject at line 23:887:
{ "revert_to": "afs_atomic_smartphone", "type": "CAMERA" }